src/AppContestMultiSeat.test.tsx

```diff
@@ -39,18 +39,14 @@ it('Single Seat Contest', async () => {
   setElectionInLocalStorage()
   setStateInLocalStorage()
 
-  const { container, getByText, queryByText } = render(<App />)
+  const { container, getAllByText, getByText, queryByText } = render(<App />)
 
   // Insert Voter Card
   currentCard = getNewVoterCard()
   advanceTimers()
 
-  // Go to Voting Instructions
-  await wait(() => fireEvent.click(getByText('Get Started')))
-  advanceTimers()
-
   // Go to First Contest
-  fireEvent.click(getByText('Start Voting'))
+  await wait(() => fireEvent.click(getAllByText('Start Voting')[1]))
```

Comment:
the `[1]` is probably because it says "start voting" somewhere else... feels fragile for the test to work this way. Is there no way to say "look for a green button that says 'start voting?'

maybe something like the `getWithinKeyboard` trick you pulled below?

Reply:
Yes, it can be done another way but I figured that someone was gonna help me find a way to remove the text in the main part of the page. 

While this may be a little fragile, it's not fragile enough to worry about at this point.